### DISCUSSION TITLE
File Upload not working on PrimeFaces version 10

### DISCUSSION THREAD

**Discussion by tcinnabon** [primefaces/community#3244] [PrimeFaces] [answered]
Hello everyone! We switched from version 6.2 to version 10, and we encountered some errors. "fileUpload" doesn't work, or more precisely its listener element

Everything is configured as indicated in your guides, but it still doesn't work. None of the solutions on the Internet helped either, so I'm writing to you.

We tried changing "commons" to "native", but it didn't help, since your native loader doesn't work in version 10.

Please suggest a possible solution to the problem

Details:
Spring Boot 3.3.4
Java 17
p.s. your forum at https://forum.primefaces.org/ is not available

**Comment by tandraschko** [comment, marked as answer]
try the newest 14.x version
there is no reason to stop on v10, the latest is always the best and most bugfree
